app/models/service_template.rb

```diff
@@ -258,6 +258,11 @@ def self.default_reconfiguration_entry_point
     nil
   end
 
+  def set_resource_actions(config_info, dialog)
```

Comment:
Thanks for the reminder.  Discussed with @bzwei before the break and I am not sure the best place for this code at the moment.

@abellotti Any suggestions on where this method should live.